GLOCKFILE

```diff
@@ -0,0 +1,19 @@
+cmd github.com/cockroachdb/c-protobuf/cmd/protoc
+cmd github.com/gogo/protobuf/protoc-gen-gogo
+cmd github.com/golang/lint/golint
+cmd github.com/robfig/glock
+cmd golang.org/x/tools/cmd/goimports
+code.google.com/p/biogo.store 183e7d0d9b67b3a25462dfe9fa518d8df377c355
+code.google.com/p/go-commander df033a4b379cd723ef7408b881c1e092cd943831
+code.google.com/p/go-uuid 35bc42037350
+code.google.com/p/snappy-go 8850bd446ad6
+github.com/agtorre/gocolorize f42b554bf7f006936130c9bb4f971afd2d87f671
+github.com/cockroachdb/c-protobuf 9e8dac59ca2a3fc82cd0665ad32b1a36f3df40b8
+github.com/cockroachdb/c-rocksdb 08ed1489772c2c06d721621eb763a44462b9741a
+github.com/coreos/etcd 0a04eec481f24ec616eed8ad52e59eb6d203263e
+github.com/gogo/protobuf bc946d07d1016848dfd2507f90f0859c9471681e
+github.com/golang/glog 44145f04b68cf362d9c4df2182967c2275eaefed
+github.com/golang/lint 39d15d55e9777df34cdffde4f406ab27fd2e60c0
+github.com/robfig/glock 78c45da050a4d993d12ad07e8ddb10a4891ac701
+golang.org/x/tools 4744be3abc70249546ce31c32fa9b5a5e96b5ad1
```

Comment:
We don't have to. I generally like the idea of moving everyone to new versions of these tools in unison, but having said that I run goimports and golint from a separate GOPATH managed by emacs so I don't get these versions. I also prefer to use glock for everything instead of leaving certain things to a `go get -u` in bootstrap.sh.